csfieldguide/templates/interactives/no-help.html

```diff
@@ -0,0 +1,19 @@
+{% extends interactive_mode_template %}
+
+{% load static %}
+
+{% block html %}
+    <div class="row interactive center-align">
+        <div class="col s12">
+            <a role="button" class="btn btn-primary btn-lg" id="interactive-no-help">Click me for help</a>
```

Comment:
I forgot to say that all text needs to be translatable. The page on Django about this is here: https://docs.djangoproject.com/en/1.11/topics/i18n/translation/

This is a big topic, so happy to talk in person about this.

Reply:
I think I managed to do this in the templates file but might need to chat to you about translation in JavaScript files